drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,77 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var entryPoint = client.get("http://example.com/").getEntryPoint();
```

Comment:
Oh, exactly. I didn't want to focus on any specific solution in this matter as there are plenty of (mentioned promises, ES17 async/await, RX, you name it). Indeed I'm using JS but duw ro fact it is juat simpler to use an existing syntax familiar for most of us than to write some abstract commands.